framework/sandstone_opts.hpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 Intel Corporation.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SANDSTONE_OPTS
+#define SANDSTONE_OPTS
+
+#include "sandstone_tests.h"
+
+struct ParsedOpts {
```

Comment:
Maybe `CommandLineOpts` would be a better name for this struct?

Reply:
`CommandLineOpts` sounds a bit raw, while they are fully parsed already.